### DISCUSSION TITLE
Covid-19 Vaccination Site

### DISCUSSION THREAD

**Discussion by nlertv** [jakobzhao/geog495#58] [Final Project]
Application : https://nlertv.github.io/covid-19vaccinationsite/
GitHub : https://github.com/nlertv/covid-19vaccinationsite

Project Description: This project is an interactive web map that features the Covid-19 Vaccination providers' location in Seattle. The main function of the interactive map is to provide more information about the vaccination site. The first part is the sidebar, in which I include the name, address, and phone number of the vaccination site. If you click on any site, it will take you to that specific site on the map. 
<img width="1419" alt="Screen Shot 2021-12-09 at 9 13 15 PM" src="https://user-images.githubusercontent.com/91760656/145522416-c97ba823-9cce-4f23-8bf5-743d825c7615.png">

**Comment by Stephenwzy** [comment]
Nice work! I like the whole layout and fresh colors of the map. It is useful to display the appointment requirement or drop-in hours in the pop-up box. Just as what you say in "Main function", it is better to add a geocoder function to search the nearest vaccination site. One improvement suggestion is that maybe you can change markers to another logo and make sure you switch the height and width so that the maker can be fully displayed.

**Comment by ghost** [comment]
The color scheme for your website is really smooth and easy on the eyes. Thematically is it very pleasing, sweet work. I also like how you displayed the address below the site name, it also creates a more clean look than what was used in the mapbox example.

My suggestion is to make the icon smaller so it doesn't bunch up so much on the map. You can re-size an icon using a re-sizer website, and then in the icon size (in JS) match the length and width to the new icon's size.

**Comment by aembar** [comment]
I like the layout and the icon you choose for the vaccination sites. The application also provides a lot of useful information for the vaccine sites such as address, phone number, and hours. Some improvements I can think of are centering the map when you load the application, it seems to be too far to the right. Also try enabling choosing multiple locations from the dropdown menu.

**Comment by NahomAgidew** [comment]
First of, I like the topic you chose. It is very practical to the current climate and very useful as an application. As an improvement, I would suggest enlarging the data source font. Additionally, when I click on each location on the sidebar, it doesn't zoom to that point (don't know if that was intentional).

**Comment by lucretiachen99** [comment]
I like your idea of mapping out these sites. It is very helpful and beneficial to people who are looking to get vaccines to stay safe. One thing I would improve on is to make the icons slightly smaller so we can see more of the map and prevent the clutter. Also it could just be me but I am unable to zoom out after I click on one location.

**Comment by mhan799** [comment]
I think this map is well-organized, straight to point, and has the appropriate theme colors. And it is very useful, especially during the pandemic. One thing that could be improved is that you can make the map abe to zoom out so users can scan over the map more easily.

**Reply by whalen323** [reply]
Cool! I like the light color scheme and being able to click the mouse function for every vaccination site, and the text is nice and easy to read. One idea I have for improvement is perhaps to use a different, more vaccine-themed marker and make them smaller so they don't crowd one another as much.

**Comment by Conibaer** [comment]
I really like the base map you chose. The color theme compliments your icons and overall design choices and works well for the information you've chosen to map. On my end there seems to be some intermittent issue with zooming although I would imagine this is probably a Map Box issue. Some easy improvements would be a slightly adjusted map initialization center and zoom to really frame up the data nicely. I had a similar problem as I didn't realize when I first started my project that the map box initialization parameters can use decimal points for increased accuracy. Adding an exit option to your popups could also come in handy

**Comment by madiframpton** [comment]
Hi! Great Work, your application provides a lot of useful information and is easy to use! I think it would be super cool if you added a search bar to your application, for the user to type in their information and have the map display the closest vaccination site. Otherwise, nicely done!

**Comment by shimizusUW** [comment]
Very helpful information, especially now, on a very well-designed map! Each marker seems to be a bit too big, clustering all of the information and location. Furthermore, I think the side panel may be too big, and the initial zoom level of the map could be changed. Once I click on a specific location, I was also unable to explore the other parts of the map.
